frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -384,6 +385,86 @@ export function ConditionsTable(props: ConditionsTableProps) {
   );
 }
 
+interface VolumeMountsProps {
+  mounts:
+    | {
+        mountPath: string;
+        name: string;
+        readOnly: boolean;
+      }[]
+    | undefined;
+}
+
+function VolumeMounts(props: VolumeMountsProps) {
+  const { mounts } = props;
+  if (!mounts) {
+    return null;
+  }
+  const classes = MetadataDisplayStyle({});
+  return (
+    <Grid container direction="column" spacing={2}>
+      {mounts?.map((item, index) => (
+        <Grid item key={`${item.name}__${index}`}>
+          <Box display="flex">
+            <Typography className={classes.metadataValueLabel}>{item.mountPath}</Typography>
+            <Box px={2}>from</Box>
+            <Typography className={classes.metadataValueLabel}>{item.name}</Typography>
+            <Box px={1}>
+              <Typography className={classes.metadataValueLabel}>
+                {item.readOnly ? 'readOnly' : 'readWrite'}
+              </Typography>
+            </Box>
```

Comment:
Boxes default to `div` AFAIK. So they shouldn't really be mixed in what reads as a sentence. Maybe using Typography with a span component is more adequate.

Reply:
Fixed